terraform/shared/shared.tf.json.template.py

```diff
@@ -19,6 +22,88 @@
         'The Cloudtrail bucket must reside in the default region',
         config.cloudtrail_s3_bucket_region, config.region)
 
+
+class CloudTrailAlarm(NamedTuple):
+    name: str
+    statistic: str
+    filter_pattern: str
+
+    @property
+    def metric_name(self) -> str:
+        return f'{self.name}_metric'
```

Comment:
```suggestion
        return self.name + '_metric'
```

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#18string-interpolation